src/applications/ivc-champva/10-7959a/config/submitTransformer.js

```diff
@@ -61,6 +61,7 @@ export default function transformForSubmit(formConfig, form) {
     copyOfData.medicalUpload,
     copyOfData.primaryEob,
     copyOfData.secondaryEob,
+    copyOfData.pharmacyUpload,
```

Comment:
This file was missing previously, so it wasn't getting attached to the generated PDF. This fixes it.